example-nextjs/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+//the argument types here needs to be fixed.
+export const runMethodOnCtrlEnterOrCmdEnterKeyPress = (
```

Comment:
This is a big nitpick, and I appreciate the descriptive method name, but it might be fine to call this `runCodeOnKeyPress` in the event that we want to support other bindings.